File: qe-selenium-java/Activity17.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.Select;

public class Activity17 {
public static void main(String[] args) {
    WebDriver driver = new FirefoxDriver();
    driver.get("https://training-support.net/webelements/selects");
    System.out.println("Title of the Page: "+driver.getTitle());

    System.out.println(driver.getTitle());

    WebElement multiSelectElement = driver.findElement(By.cssSelector(".h-80"));

    // Create Select object
    Select multiSelect = new Select(multiSelectElement);

    multiSelect.selectByVisibleText("HTML");

    multiSelect.selectByIndex(4);
    multiSelect.selectByIndex(5);
    multiSelect.selectByIndex(6);
    multiSelect.deselectByIndex(5);

    // Close browser
    driver.quit();

    }
}
    

File: qe-selenium-python/Activity17.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import Select


with webdriver.Firefox() as driver:

    driver.get("https://training-support.net/webelements/selects")

    print("Title of the Page:", driver.title)
    print(driver.title)

    multi_select_element = driver.find_element(By.CSS_SELECTOR, ".h-80")

    # Create Select object
    multi_select = Select(multi_select_element)

    # Select options
    multi_select.select_by_visible_text("HTML")

    multi_select.select_by_index(4)
    multi_select.select_by_index(5)
    multi_select.select_by_index(6)

    # Deselect one option
    multi_select.deselect_by_index(5)
